lib/cdo/chat_client.rb

```diff
@@ -17,7 +17,7 @@ def self.developers(message, options={})
   end
 
   def self.log(message, options={})
-    message(CDO.hip_chat_log_room, message, options)
```

Comment:
TODO(asher): Remove `CDO.hip_chat_log_room`, `CDO.hip_chat_logging`, and `Slack.CHANNEL_MAP`.